docs/src/inverse_problems/structural_identifiability.md

```diff
@@ -0,0 +1,168 @@
+
+### Indicating known parameters
+In the previous case we assumed that all parameters are unknown, however, this is not necessarily true. If there are parameters with known values, we can supply these using the `known_p` argument. Providing this additional information might also make other, previously unidentifiable, parameters identifiable. Let us consider the previous example, where we measure the concentration of $M$ only, but now assume we also know the production rate of $E$ ($pₑ$):
+```example si1
+assess_identifiability(gwo; measured_quantities=[:M], known_p=[:pₑ], loglevel=Logging.Error)
+```
+Not only does this turn the previously non-identifiable `pₑ` (globally) identifiable (which is obvious, given that its value is now known), but this additional information improve identifiability for several other network components.
+
+To, in a similar manner, indicate that certain initial conditions are known is a work in progress. Hopefully the feature should be an available in the near future.
+
+### Providing non-trivial measured quantities
+Sometimes, we are not actually measuring species species, but rather some combinations of species (or possibly parameters). To account for this, `measured_quantities` accepts any algebraic expressions (and not just single species). To form such expressions, species and parameters have to first be `@unpack`'ed from the model. Say that we have a model where an enzyme ($E$) is converted between an active and inactive form, which in turns activates the production of a product, $P$:
+```example si2
+using Catalyst, StructuralIdentifiability # hide
+enzyme_activation = @reaction_network begin
+    (kA,kD), Eᵢ <--> Eₐ
+    (Eₐ, d), 0 <-->P
+end
+```
+and we can measure the total amount of $E$ ($=Eᵢ+Eₐ$), as well as the amount of $P$, we can use the following to assess identifiability:
+```example si2
+@unpack Eᵢ, Eₐ = enzyme_activation
+assess_identifiability(enzyme_activation; measured_quantities=[Eᵢ+Eₐ, :P], loglevel=Logging.Error)
+nothing # hide
+```
+
+### Assessing identifiability for specified quantities only
+By default, StructuralIdentifiability assesses identifiability for all parameters and variables. It is, however, possible to designate precisely which quantities you want to check using the `funcs_to_check` option. This both includes selecting a smaller subset of parameters and variables to check, or defining customised expressions. Let us consider the Goodwind from previously, and say that we would like to check whether the production parameters ($pₘ$, $pₑ$, and $pₚ$) and the total amount of the three species ($P + M + E$) are identifiable quantities. Here, we would first unpack these (allowing us to form algebraic expressions) and then use the following code:
+```example si1
+@unpack pₘ, pₑ, pₚ, M, E, P = goodwind_oscillator
+assess_identifiability(goodwind_oscillator; measured_quantities=[:M], funcs_to_check=[pₘ, pₑ, pₚ, M + E + P], loglevel=Logging.Error)
```

Comment:
Perhaps `quantities_to_identify` or `quants_to_ident` would be clearer for users / more descriptive than `funcs_to_check`?

Reply:
Agreed, but again, this is what they sue in StructuralIdentifiability. I can use something else (like for `p`), but figured it would make sense to use what they use.